machine_learning/decision_tree_churn/churn_cal.py

```diff
@@ -0,0 +1,55 @@
+import doctest
+
+import numpy as np
+import pandas as pd
+from sklearn import model_selection, tree
+
+
+def calculate_churn_rate() -> np.ndarray:
+    """
+    Calculates the churn rate of customers using a decision tree classifier.
+
+    >>> calculate_churn_rate()
+    array([0, 0, 0, 0, 1])
+    """
+
+    churn_md = pd.read_csv("churn_modelling.csv")
```

Comment:
Enable experimentation with multiple files that are in the same directory as this file...

```suggestion
def calculate_churn_rate(filename: str = "churn_modelling.csv") -> np.ndarray:
    """
    Calculates the churn rate of customers using a decision tree classifier.

    >>> calculate_churn_rate()
    array([0, 0, 0, 0, 1])
    """

    churn_md = pd.read_csv(Path(__file__).with_name(filename).absolute())
```